src/libsystemd/sd-bus/bus-dump.c

```diff
@@ -81,6 +90,8 @@ _public_ int sd_bus_message_dump(sd_bus_message *m, FILE *f, uint64_t flags) {
                 if (m->reply_cookie != 0)
                         fprintf(f, "  ReplyCookie=%" PRIu64, m->reply_cookie);
 
+                fprintf(f, "  Timestamp=\"%s\"", p ? p : "unrepresentable");
```

Comment:
`strna(p)`.

Reply:
haven't look for strna() in the last commit.
will fix that in the next commit

Reply:
fixed with 729c784